Toggl.Core/DataSources/SingletonDataSource.cs

```diff
@@ -66,6 +66,13 @@ public override IObservable<TThreadsafe> Update(TThreadsafe entity)
                 .SingleAsync()
                 .Do(handleConflictResolutionResult);
 
+        public void ReportChange()
+        {
+            // .Wait() is ugly, but we won't be blocking the thread for very long
+            // because we're not doing any intensive task, we're just calling OnNext
+            Get().Do(currentSubject.OnNext).Wait();
```

Comment:
```suggestion
            Get().Do(currentSubject.OnNext).FirstAsync().GetAwaiter().GetResult();
```
I'd prefer this way of unwrapping the result, mostly for better error logging (and the fact that we consistently used the same way in the app already.

[Some reasoning behind this.](https://stackoverflow.com/questions/36426937/what-is-the-difference-between-wait-vs-getawaiter-getresult)